src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -58,11 +59,18 @@ public Task GetLicenseConfigurationSettings()
             using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
             using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
             {
+                var canConfigLicense = true;
+                var authentication = HttpContext.Features.Get<IHttpAuthenticationFeature>() as RavenServer.AuthenticateConnection;
+                if (authentication != null && authentication.Status != RavenServer.AuthenticationStatus.ClusterAdmin)
+                {
+                    canConfigLicense = false;
+                }
+                
                 var djv = new DynamicJsonValue
                 {
-                    [nameof(LicenseConfiguration.CanRenew)] = ServerStore.Configuration.Licensing.CanRenew,
-                    [nameof(LicenseConfiguration.CanActivate)] = ServerStore.Configuration.Licensing.CanActivate,
-                    [nameof(LicenseConfiguration.CanForceUpdate)] = ServerStore.Configuration.Licensing.CanForceUpdate
+                    [nameof(LicenseConfiguration.CanRenew)] = ServerStore.Configuration.Licensing.CanRenew && canConfigLicense,
```

Comment:
Wouldn't it be better to get auth status from elsewhere? It would give you more flexibility in the UI e.g.
License can be renewed, but you are not auth as Cluster Admin is a different case than License cannot be renewed at all.

Reply:
done